packages/checkbox/tsconfig.json

```diff
@@ -3,5 +3,16 @@
   "compilerOptions": {
     "declarationDir": "dist"
   },
-  "include": ["src/**/*", "../../typings/*"]
+  "include": ["src/**/*", "../../typings/*"],
+  "references": [
```

Comment:
Doing this for every package feels like a lot. 🤔

Reply:
Also a shame that the references array doesn't also accept path strings rather than objects.

Reply:
Could we have the references array only in `package.tsconfig.json`? That could be a whole lot cleaner if so.

Reply:
Yeah I agree on all counts. I'm not happy about any of it.

AFAICT the way references works is it essentially tells tsc what the dependencies are so it references them and builds them before building the current project. 

We can definitely play with it a little more but this was me getting things into a working state. We can now try optimizing it.